### PR TITLE
fix e2e issues

### PR DESCRIPTION
**Pull request by zhujian7**

### CODE REVIEW

File: build/run-e2e-tests.sh [hunk 1]
```diff
@@ -75,12 +75,22 @@ ${KUBECTL} --context="${cluster_context}" -n kube-system scale --replicas=1 depl
...
-echo "###### deploy registration-operator"
-rm -rf "$WORK_DIR/registration-operator"
-git clone https://github.com/stolostron/registration-operator.git "$WORK_DIR/registration-operator"
-${KUBECTL} apply -k "$WORK_DIR/registration-operator/deploy/cluster-manager/config/manifests"
-${KUBECTL} apply -k "$WORK_DIR/registration-operator/deploy/cluster-manager/config/samples"
-rm -rf "$WORK_DIR/registration-operator"
+echo "###### deploy ocm"
+rm -rf "$WORK_DIR/_repo_ocm"
+
+export OCM_VERSION=main
```
Comment (xuezhaojun) [inline build/run-e2e-tests.sh:81]: Do we need to use `latest` here?

Comment (zhujian7) [reply build/run-e2e-tests.sh:81]: `main` and `latest` are the same image.

Comment (zhujian7) [reply build/run-e2e-tests.sh:81]: https://github.com/open-cluster-management-io/ocm/pull/214 this PR is trying to fix the e2e issue.

Comment (xuezhaojun) [reply build/run-e2e-tests.sh:81]: Got it.

**Pull request merged**